src/debug/debug.cpp

```diff
@@ -988,43 +988,67 @@ static void SetCodeWinStart()
 /*    User input    */
 /********************/
 
-uint32_t GetHexValue(char* str, char*& hex)
+// Looks for the prefix string at the start of the initial string,
+// doing a case-insensitive compare. If it is found, consumes the
+// matching prefix and returns true. Otherwise, returns false and
+// leaves the string unchanged.
+static bool ConsumePrefixNoCase(char*& str, std::string_view prefix)
```

Comment:
I know it's inconsistent in older code, but for new static functions we should use `lower_snake_case` as per our convention.